app/views/state_file/questions/nj_estimated_tax_payments/edit.html.erb

```diff
@@ -0,0 +1,35 @@
+<%
+  title = t(".title", filing_year: @filing_year)
+  content_for :page_title, title
+%>
+
+<% content_for :card do %>
+  <h1 class="h2"><%= t(".title", filing_year: @filing_year) %></h1>
+
+  <p><%= t(".subtitle") %></p>
+  <ul class="list--bulleted spacing-below-25">
+    <li><%= t(".subtitle_list_1") %></li>
+    <li>
+      <%= t(".subtitle_list_2", filing_year: @filing_year, prior_year: @prior_year) %>
+      <ul class="list--bulleted">
+        <li><%= t(".subtitle_list_2_sub", prior_year: @prior_year) %></li>
+      </ul>
+    </li>
+  </ul>
+
+  <%= form_with model: @form, url: { action: :update }, local: true, method: "put", builder: VitaMinFormBuilder do |f| %>
+    <div class="white-group">
+      <%= f.vita_min_money_field(
+        :estimated_tax_payments,
+        t('.label'),
+        classes: ["form-width--long"],
+        options: { placeholder: "0" }
```

Comment:
[q] the placeholder default is `0.00`, is this a design choice on NJ team to use `0` instead?

Reply:
It is, Hannah could speak to that further